stable/grafana/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "grafana.fullname" . }}
+  labels:
+    app: {{ template "grafana.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+    apparmor.security.beta.kubernetes.io/allowedProfileNames: 'runtime/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+    apparmor.security.beta.kubernetes.io/defaultProfileName:  'runtime/default'
+spec:
+  privileged: false
+  allowPrivilegeEscalation: false
+  allowedCapabilities:
+    - CAP_CHOWN
```

Comment:
This and the 'RunAsAny' on line 32 are [only needed for images prior to 5.1.0](https://hub.docker.com/r/grafana/grafana/). I could condition on the image tag to render the most restrictive PSP. I'm not sure if that's needed/desired.

What's the typical approach, here?

Reply:
Personally, I'd move forward with the chart and not make it backwards-compatible.

@zanhsieh @rtluckie What do you think?

Reply:
Agreed with @unguiculus . I think we might add `kubeVersion` in Chart.yaml file as [charts.md](https://github.com/kubernetes/helm/blob/master/docs/charts.md#the-chartyaml-file) suggested.